### PR TITLE
Cleaned tests code

### PR DESCRIPTION
**Pull request by xavismeh**
Refactoring + coding standards

Note that this PR is "blind": no tests suite has been run.

### CODE REVIEW

File: tests/Integration/EmptyArraysHandlingTest.php [hunk 1]
```diff
@@ -22,9 +23,11 @@ public function testEmptyArrayAsListIsHandled()
         MERGE (f)-[:KNOWS]->(n)';
 
         $params = ['id' => 'me', 'friends' => Collections::asList([])];
-        $this->getSession()->run($query, $params);
 
-        $result = $this->getSession()->run('MATCH (n:User) RETURN count(n) AS c');
+        $session = $this->getSession();
```
Comment (ikwattro) [inline tests/Integration/EmptyArraysHandlingTest.php:27]: This goes in contradiction to https://github.com/graphaware/neo4j-bolt-php/pull/24/files#diff-0c380b4ddfb5d56023500d4614d375bdR44

**Review by ikwattro — CHANGES_REQUESTED** [review]
Please remove phpstorm files from the PR

File: tests/Integration/EmptyArraysHandlingTest.php (hunk 1 above)
Comment (xavismeh) [reply tests/Integration/EmptyArraysHandlingTest.php:27]: In this case, the session is used multiple times when the file you're referring to it's not the case ;)

**Comment by xavismeh** [comment]
Done @ikwattro :)

**Comment by ikwattro** [comment]
@xavismeh I still see .idea/php.xml

**Comment by xavismeh** [comment]
Sorry for this, I didn't notice. That's removed now :)

**Comment by ikwattro** [comment]
Thx

**Pull request merged**